redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -226,7 +310,7 @@ def __init__(self,
                  normalization_type: str,
                  weight_sharing: bool = True,
                  ignore_item_embedding: bool = False,
-                 save_directory_path: str = None) -> None:
+                 save_directory_path: str = None):
```

Comment:
@mski-iksm 
Why did you delete this type hinting?

Reply:
It's a mistake. I've fixed it.